### ISSUE TITLE
With VoiceOver on TWTRTweetTableViewCell focuses on an invisible line

### ISSUE THREAD

**Issue by lorenzOliveto** [twitter/twitter-kit-ios#34]
With VoiceOver on `TWTRTweetTableViewCell` focuses on an invisible line after the text of the tweet and the user cannot advance after it with the standard swipe gesture. 

This is the code used for the cell initialization:

```
- (UITableViewCell *) tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
    TWTRTweetTableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:@"TWTRTweetTableViewCell.h"];
    if (cell == nil){
        cell = [[TWTRTweetTableViewCell alloc] initWithStyle:UITableViewCellStyleDefault reuseIdentifier:@"TWTRTweetTableViewCell.h"];
    }
    TWTRTweet *tweet = [tweets objectAtIndex:indexPath.item];
    [cell configureWithTweet:tweet];
    cell.selectionStyle = UITableViewCellSelectionStyleNone;
    return cell;
}
```

iOS Version: 11.2.6
iOS Device Model: iPhone SE
TwitterKit Version Number: 3.3.0
TwitterCore Version Number: 3.1.0

![simulator screen shot - iphone 8 - 2018-02-28 at 16 32 49](https://user-images.githubusercontent.com/3470464/36796265-739ca44a-1ca5-11e8-9547-a809e4aec33f.png)

**Comment by katejlee** [comment]
Unfortunately, with recent deprecation, we don't anticipate any more bug fixes or feature development unless it's breaking change. Feel free to take a look at the github repository and fork out to make changes! Thanks!
